source/nordic_sdk/components/ble/common/ble_date_time.h

```diff
@@ -30,8 +30,8 @@
  *
  */
 
-/* Attention! 

```

Comment:
this file has only white space diffs. We should remove it from the pull request. Let's avoid un-necessary changes.